src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -2170,6 +2244,10 @@ JNIEXPORT void JNICALL Java_sun_awt_windows_WPrinterJob_frameRect
 
   TRY;
 
+    if ((HDC)printDC == NULL) {
+         return;
+     }
```

Comment:
```suggestion
  if ((HDC)printDC == NULL) {
    return;
  }
```

There are several cases of similar alignment issues.